BuildResidentialHPXML/measure.rb

```diff
@@ -3037,6 +3050,10 @@ def validate_arguments(args)
       errors << "foundation_wall_insulation_distance_to_bottom=#{args[:foundation_wall_insulation_distance_to_bottom]} and geometry_foundation_height=#{args[:geometry_foundation_height]}" if error
     end
 
+    # ambient with rim joists
+    error = (args[:geometry_foundation_type] == HPXML::FoundationTypeAmbient) && (args[:geometry_rim_joist_height] > 0)
+    errors << "geometry_foundation_type=#{args[:geometry_foundation_type]} and geometry_rim_joist_height=#{args[:geometry_rim_joist_height]}" if error
```

Comment:
It also won't apply to a slab foundation type. But I'm tempted to say not to generate an error here and just ignore it if it doesn't apply (especially if we have a default > 0).